README.md

```diff
@@ -1,15 +1,64 @@
-**Project Title:**  Ekklesia
+### About
+Project Title:   Ekklesia
+Built By:  Benjamin A. Ngafua
+Location: Monrovia, Liberia
+Description: Church Management Software
+Video Demo: [Benjamin A. Ngafua's cs50x Final Project](https://youtu.be/nzqE3m4DkJ8)
 
-**Built By:** Benjamin A. Ngafua
+### What is Ekklesia app?
+- Ekklesia is a church management software that aids churches and similar in managing, automating and organizing daily operations.
 
-**City/Country:** Monrovia, Liberia
+### Ekklesia aim?     
+- It allows church members to see God's hand through ministries
+
+### What it is used for?
+1. It tracks and manages resources
+2. It enhances communication
+3. It monitors church growth
+
+### Who to use it currently?
+ - Currently use for administrator.
+
+### Technology used
+- **Framework:**
+    - Flask
+    
+- **Templates :**
+    - Jinja2
+- **Frontend**
+    - >Template from: [TEMPLATEMO](https://themewagon.com/)
+
+- **Backend Language:** 
+    - Python
+- **Database**
+    - SQLite3
+- **Images use:**
+    - > The images are from: [PEXELS](https://www.pexels.com/)
+
+
+### Feature
+|Description |Status |
+|---------|------|
+|Dashboard statistics | 80%|
+| Store users | 80% |
+|Create Members | 80% |
+|Communicate to members| Yet to start|
+| Track financial transaction | Yet to start |
+
+### Setup
+Install all the modules in requirements.txt file manually or 
+
+Run 
+> ```pip3 freeze > requirements.txt``` 
+
+To run the app locally run
+>```flask run```
```

Comment:
This is not working as well. I guess it is because of some packages that aren't installed.
![image](https://user-images.githubusercontent.com/20514920/187154165-59dbd312-d1da-4472-b800-a2e95f8a06e4.png)